Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -86,7 +90,13 @@ def bump_formula_pr
 
     old_formula_version = formula_version(formula, requested_spec)
 
-    replacement_pairs = if new_url_hash
+    replacement_pairs = if requested_spec == :stable && formula.revision != 0
+      [[/^  revision \d+\n/m, ""]]
+    else
+      []
+    end
```

Comment:
Hehe. I feel like that's something I would write, and then you'd request it be replaced with what's in the PR right now!